easybuild/easyconfigs/g/GROMACS/GROMACS-2024.2-foss-2023b-CUDA-12.5.0.eb

```diff
@@ -74,7 +74,8 @@ exts_default_options = {
 
 exts_list = [
     ('gmxapi', '0.4.2', {
-        'preinstallopts': 'export CMAKE_ARGS="-Dgmxapi_ROOT=%(installdir)s -C %(installdir)s/share/cmake/gromacs_mpi/gromacs-hints_mpi.cmake" && ',
+        'preinstallopts': 'export CMAKE_ARGS="-Dgmxapi_ROOT=%(installdir)s ' +
+                          '-C %(installdir)s/share/cmake/gromacs_mpi/gromacs-hints_mpi.cmake" && ',
         'source_tmpl': 'gromacs-2023.3.tar.gz',
```

Comment:
This should correspond to the current version.

Reply:
Ugh... Fixed in 5dc10572a0

Reply:
@boegel what about the gmxapi version? I'm not completely sure what the correct version number should be. https://gitlab.com/gromacs/gromacs/-/blob/v2024.2/python_packaging/gmxapi/pyproject.toml?ref_type=tags has 0.4.1 (?!), but https://gitlab.com/gromacs/gromacs/-/blob/v2024.2/python_packaging/gmxapi/src/gmxapi/version.py?ref_type=tags and https://manual.gromacs.org/2024.2/gmxapi/index.html say 0.5.0a1. So I guess the latter??